massa-execution-worker/src/execution.rs

```diff
@@ -197,7 +197,7 @@ impl ExecutionState {
             #[cfg(feature = "execution-trace")]
             trace_history: Arc::new(RwLock::new(TraceHistory::new(
                 config.max_execution_traces_slot_limit as u32,
-                MAX_OPERATIONS_PER_BLOCK,
+                (MAX_GAS_PER_BLOCK / BASE_OPERATION_GAS_COST) as u32,
```

Comment:
You updated this because we are currently gas limited, but I propose to put the min value between MAX_OPERATIONS_PER_BLOCK and (MAX_GAS_PER_BLOCK / BASE_OPERATION_GAS_COST) as u32,

Reply:
done